### ISSUE TITLE
11.1.4

### ISSUE THREAD

**Issue by goetsu** [DISIC/rgaa_referentiel#1]
Test 11.1.4 : Chaque champ de formulaire qui utilise une propriété ARIA aria-label doit être accompagné d’un passage de texte visible et accolé au champ permettant de comprendre la nature de la saisie attendue. Cette règle est-elle respectée ?

---

Problème

Les techniques ARIA6 et ARIA14 référencées ne demandent nullement la présence d'un élément visible. La technique ARIA6 vérifie la pertinence et la technique ARIA14, au contraire de ce que demande ce test, autorise explicitement l'emploi d'un aria label quand il n'est pas possible d'avoir un élément visible "Using aria-label to provide an invisible label where a visible label cannot be used"

Par ailleurs, si le RGAA décide effectivement d'interdire cela malgré que ce soit autorisé par les techniques wcag à minima il faut autoriser le title comme solution puisqu'il est déjà autorisé comme technique pour "labelliser" un champ

**Comment by goetsu** [comment]
Dans le même ordre d'idée je ne vois pas pourquoi un label masqué à l'écran serait dans ce cas une solution autorisée alors le aria-label ne l'est pas

**Comment by Etat-Plateforme** [comment]
Merci pour votre commentaire.

RGAA n’est pas plus restrictif que WCAG mais prend en charge une situation où un label de champ de formulaire utilise la seule propriété `aria-label`. Dans ce cas, parce que la propriété `aria-label` n’est pas visible il peut y avoir un risque que le champ de formulaire ne soit pas compris par certains utilisateurs.

Néanmoins vous avez raison sur le fait qu’obliger à ce qu’un texte visible soit systématiquement proposé dans le cas de l’utilisation d’un `aria-label` est trop restrictif.

Le test a été reformulé en ajoutant « si nécessaire » de manière à pouvoir l’adapter au contexte.

Le test 11.1.4 a été corrigé en conséquence. Ces modifications sont indexées dans la note de révision.

Nous vous renvoyons également à l’issue #12 qui traite d’un problème similaire.
